pyradiosky/tests/test_skymodel.py

```diff
@@ -902,63 +710,3 @@ def test_array_to_skymodel_loop():
 
     assert np.allclose((sky.ra - sky2.ra).rad, 0.0)
     assert np.allclose((sky.dec - sky2.dec).rad, 0.0)
-
-
-class TestMoon:
```

Comment:
Why did you remove this test class?

Reply:
Not sure how that happened... it's back now.